### DISCUSSION TITLE
need help with createSelector and TypeScript

### DISCUSSION THREAD

**Discussion by EdvardSinkevichTR** [reduxjs/redux-toolkit#1998] [Q&A]
![image](https://user-images.githubusercontent.com/85125134/152578162-dee00487-fc2e-4503-a7cb-d881c2b06b82.png)
I see the next error
![image](https://user-images.githubusercontent.com/85125134/152578270-b1f40948-6570-4c8c-9ac9-3107e9ffb6d3.png)
 store config
 
![image](https://user-images.githubusercontent.com/85125134/152578433-587f174c-0d7e-49de-b82f-3168f0fb142d.png)

similar issue on stackoverflow
https://stackoverflow.com/questions/70923750/is-it-possible-to-export-a-createselector-variable-in-typescript

**Comment by phryneas** [comment]
If you don't need it, disable `declaration: true` in your `tsconfig.json`. If you need that (only libraries do), you probably need to add `redux` as a dependency or devdependency.
